app/views/layouts/_footer.html.erb

```diff
@@ -45,7 +45,7 @@
           <li><%= link_to ts('otwarchive %{rev_number}', :rev_number => ArchiveConfig.REVISION), "https://github.com/otwcode/otwarchive/commits/#{ArchiveConfig.REVISION}" %></li>
         <% end %>
         <li><%= link_to ts('Known Issues', key: 'footer'), known_issues_path %></li>
-        <li><a href="http://www.gnu.org/licenses/gpl-2.0.html" title="<%= ts('View License', key: 'footer') %>">GPL</a> <%= ts('by the', key: 'footer') %> <a href="http://transformativeworks.org/" title="<%= ts('The Organization for Transformative Works', key: 'footer') %>"><%= ts('OTW', key: 'footer') %></a></li>
+        <li><a href="https://www.gnu.org/licenses/gpl-2.0.html" title="<%= ts('View License', key: 'footer') %>">GPL</a> <%= ts('by the', key: 'footer') %> <a href="https://transformativeworks.org/" title="<%= ts('The Organization for Transformative Works', key: 'footer') %>"><%= ts('OTW', key: 'footer') %></a></li>
```

Comment:
Since this link redirects to https://www.gnu.org/licenses/old-licenses/gpl-2.0.html, can you change to https://www.gnu.org/licenses/old-licenses/gpl-2.0.html instead?